### PR TITLE
feat(AHWR-202): Define internal contract for messages to SFD Messaging Proxy, crn and sbi fields added

### PR DESCRIPTION
**Pull request by bendef-24**

### CODE REVIEW

File: app/config/index.js [hunk 1]
```diff
@@ -25,6 +30,10 @@ const config = {
   claimServiceUri: process.env.CLAIM_SERVICE_URI,
   endemics: {
     enabled: process.env.ENDEMICS_ENABLED
+  },
+  sfdRequestMsgType: `${msgTypePrefix}.sfd.request`,
```
[the remark this answers is not in the document]
Comment (bendef-24) [reply app/config/index.js:34]: Agreed, will do

File: app/email/notify-send.js [hunk 2]
```diff
@@ -11,10 +11,19 @@ const {
 } = require('../config').notifyConfig
 const { update } = require('../repositories/document-log-repository')
 const appInsights = require('applicationinsights')
+const sendSFDEmail = require('./sfd-client')
+const { sfdMessage } = require('../config')
 
 const send = async (templateId, email, personalisation) => {
   console.log(`Received email to send to ${email} for ${personalisation.reference}`)
   try {
+    if (sfdMessage.enabled) {
```
[the remark this answers is not in the document]
Comment (bendef-24) [reply app/email/notify-send.js:20]: Agreed, not ideal but happy to refactor in next iteration

File: app/email/sfd-client.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const sendMessage = require('../messaging/send-message')
+const { sfdRequestMsgType, messageQueueConfig: { sfdMessageQueue } } = require('../config')
+const validateSFDClaim = require('../messaging/submit-sfd-schema')
+const states = require('../messaging/states')
+
+const sendSFDEmail = async (templateId, email, personalisation) => {
+  const { personalisation: { reference, crn, sbi } } = personalisation
```
[the remark this answers is not in the document]
Comment (bendef-24) [reply app/email/sfd-client.js:7]: Agreed, looking to fix that now

File: app/config/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (omega1119) [reply app/config/index.js:34]: It's the pattern from the application service. The suffixes are different for each message type.

**Review by omega1119 — DISMISSED** [review]

**Review by aaroncarroll82 — APPROVED** [review]

**Comment by aaroncarroll82** [comment]
The CI Jenkins check is marked as red and blockign merge, however it's not pointing to the latest, which is green, so I will bypass that in order to conclude merge.

**Pull request merged**